stroom-core-server/src/main/java/stroom/connectors/ExternalLibService.java

```diff
@@ -0,0 +1,90 @@
+package stroom.connectors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+import stroom.node.server.StroomPropertyService;
+import stroom.util.spring.StroomScope;
+
+import javax.inject.Inject;
+import java.io.File;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * This service can be used by any factory services that load in external libraries.
+ *
+ * It effectively wraps the {@link ServiceLoader} load function. Using a property in stroom.conf
+ * to locate and load external JAR files with separate class loaders.
+ */
```

Comment:
This class basically wraps the URL classloaders, using the named property to find the JAR files.